LayoutTests/accessibility/date-input-aria-label-and-value.html

```diff
@@ -0,0 +1,100 @@
+                let result = await expectAsync(`axDateFieldWrapper.childAtIndex(${childIndex}).intValue`, `${description}`);
+
+                result += await expectAsync(`axDateFieldWrapper.childAtIndex(${childIndex}).valueDescription`, `'AXValueDescription: ${description}'`);
+
+                return result;
+            }
+            
+            async function expectChildToHaveARIALabel(childIndex, labelText) {
+                return expectAsync(`platformValueForW3CName(axDateFieldWrapper.childAtIndex(${childIndex}))`, labelText);
+            }
+            
+            const dateInput = document.getElementById("date-input");
+
+            outputString('Checking ARIA Labels for sub-fields');
+            testOutput += await expectChildToHaveARIALabel(0, `'month'`);
+            testOutput += await expectChildToHaveARIALabel(1, `'day'`);
+            testOutput += await expectChildToHaveARIALabel(2, `'year'`);
+            
+            outputString("Focusing date element and pressing up arrow to increment the first field.");
+            dateInput.focus();
+            keyDown("upArrow");
+
+            testOutput += await expectChildToHaveARIAValueAttributes(0, 3);
+
+            keyDown("downArrow", 2, "Simulating down arrow press to decrement the first field.");
+            testOutput += await expectChildToHaveARIAValueAttributes(0, 1);
+
+            outputString("Simulating tab key to move to the next field in the date input.");
+            keyDown("\t");
+            testOutput += await expectChildToHaveARIAValueAttributes(1, 2);
+
+            outputString('Simulating typing "10" in the second input field.');
+            keyDown("1");
+            keyDown("0")
+            testOutput += await expectChildToHaveARIAValueAttributes(1, 10);
+
+            outputString("Simulating down arrow press to decrement the second field value.")
+            keyDown("downArrow");
+            testOutput += await expectChildToHaveARIAValueAttributes(1, 9);
+
+            keyDown("upArrow", 2, "Simulating up arrow press to increment the second field value.");
+            testOutput += await expectChildToHaveARIAValueAttributes(1, 11);
+
+            outputString("Simulating tab key to move to the next field in the date input.");
+            keyDown("\t");
+            testOutput += await expectChildToHaveARIAValueAttributes(2, 2022);
+
+            keyDown("downArrow", 10, "Simulating down arrow press to decrement the third field value.");
+            testOutput += await expectChildToHaveARIAValueAttributes(2, 2012);
+
+            outputString("Simulating up arrow press to increment the third field value.")
+            keyDown("upArrow");
+            testOutput += await expectChildToHaveARIAValueAttributes(2, 2013);
+
+            outputString("Manually setting the date input's value via JavaScript.")
+            dateInput.value = '2023-12-30';
```

Comment:
This test mostly uses double quotes, but occasionally uses single quotes (like this spot). It would be nice to use double quotes consistently everywhere.